BUILD

```diff
@@ -64,11 +64,11 @@ config_setting(
 )
 
 # This should be updated along with build.yaml
-g_stands_for = "gold"
+g_stands_for = "godric"
 
-core_version = "7.0.0-dev"
+core_version = "7.0.0"
```

Comment:
I saw the PR comment explaining the rationale for not using the  suffix -dev, -pre1 etc for core version and I sort of agree with the rationale. However should there be a discussion before changing the practice to see if there was a reason?

Reply:
I had discussed this with @nicolasnoble some time ago. Since Core API doesn't have a release process, we are just cycling through 7.0.0, 7.0.0-dev, 7.0.0-pre1 and back to 7.0.0 which is simply wrong. Increasing the version without any changes to Core API doesn't make sense either.